### PR TITLE
Update http-behaviors.md

### PR DESCRIPTION
**Pull request by cmanhard**
PI Web API can return 404, 409, and 501.  We should list them as possible.

### CODE REVIEW

File: documentation/http-behaviors.md [hunk 1]
```diff
@@ -22,6 +23,9 @@ The following status codes are returned by OSIsoft products accepting OMF messag
 | `400 Bad Request` | The OMF message was malformed or not understood. The client should not retry sending the message without modifications. |
 | `401 Unauthorized` | Authentication failed. |
 | `403 Forbidden` | Authentication succeeded, but not authorized. |
+| `404 Not Found` | A referenced resource was not found. |
+| `409 Conflict` | The OMF message contains a conflict with an existing destination resource. |
 | `413 Payload Too Large` | Payload size exceeds OMF body size limit. |
 | `500 Internal Server Error` | The server encountered an unexpected condition. |
-| `503 Service Unavailable` | The server is currently unavailable, retry later. |
\ No newline at end of file
+| `501 Not Implemented` | A feature requested is not available on the end point. Generally, unsupported OMF features are ignored; however, when an operation type or a specific aspect of an OMF resource cannot be respresented, this error may be returned. |
```
Comment (zryska) [inline documentation/http-behaviors.md:30]: Minor typo => respresented => represented.

File: documentation/http-behaviors.md [hunk 2]
```diff
@@ -12,7 +12,8 @@ of `413 Payload Too Large`. Compression is highly recommended for optimal perfor
 
 ## HTTP Status Codes
 
-The following status codes are returned by OSIsoft products accepting OMF messages over HTTP.
+The following status codes are possible to be returned by OSIsoft products accepting OMF messages over HTTP. 
+Not all end points will respond with the same error codes due to asyncrounous message processing.
```
Comment (zryska) [inline documentation/http-behaviors.md:16]: Small typo - asyncrounous => asynchronous

Comment (cmanhard) [reply documentation/http-behaviors.md]: Fixed.  Also added 201.

File: documentation/http-behaviors.md (hunk 1 above)
Comment (cmanhard) [reply documentation/http-behaviors.md]: fixed.

**Review by zryska — APPROVED** [review]

**Comment by cmanhard** [comment]
Discussed in architecture as well.

**Pull request merged**